packages/nimble-blazor/NimbleBlazor/Components/NimbleTableColumnAnchor.razor.cs

```diff
@@ -0,0 +1,98 @@
+    /// </summary>
+    [Parameter]
+    [DisallowNull]
+    public string HrefFieldName { get; set; } = null!;
+
+    /// <summary>
+    /// The text to show when no label value is available for a particular cell in the column of a <see cref="NimbleTable{TData}"/>
+    /// </summary>
+    [Parameter]
+    public string? Placeholder { get; set; }
+
+    /// <summary>
+    /// Hints at the human language of the linked URL. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a for more information.
+    /// </summary>
+    [Parameter]
+    public string? HrefLang { get; set; }
+
+    /// <summary>
+    /// A space-separated list of URLs. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a for more information.
+    /// </summary>
+    [Parameter]
+    public string? Ping { get; set; }
+
+    /// <summary>
+    /// How much of the referrer to send when following the link. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a for more information.
+    /// </summary>
+    [Parameter]
+    public string? ReferrerPolicy { get; set; }
+
+    /// <summary>
+    /// The relationship of the linked URL as space-separated link types. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a for more information.
+    /// </summary>
+    [Parameter]
+    public string? Rel { get; set; }
+
+    /// <summary>
+    /// Where to display the linked URL, as the name for a browsing context. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a for more information.
+    /// </summary>
+    [Parameter]
+    public string? Target { get; set; }
+
+    /// <summary>
+    /// Hints at the linked URL's format with a MIME type. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a for more information.
+    /// </summary>
+    [Parameter]
+    public string? Type { get; set; }
+
+    /// <summary>
+    /// Causes the browser to treat the linked URL as a download. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a for more information.
+    /// </summary>
+    [Parameter]
+    public string? Download { get; set; }
+
+    /// <summary>
+    /// The fractional/proportional width to use for this column
+    /// </summary>
+    [Parameter]
+    public double FractionalWidth { get; set; } = 1;
+
+    /// <summary>
+    /// The minimum width (in pixels) for this column
+    /// </summary>
+    [Parameter]
+    public double? MinPixelWidth { get; set; }
+
+    /// <summary>
+    /// Specifies the grouping precedence of the column within the set of all columns participating in grouping.
+    /// Columns are rendered in the grouping tree from lowest group-index as the tree root to highest
+    /// group-index as tree leaves.
+    /// </summary>
+    [Parameter]
+    public int? GroupIndex { get; set; }
+
+    /// <summary>
+    /// Whether or not this column can be used to group rows by
+    /// </summary>
+    [Parameter]
+    public bool? GroupingDisabled { get; set; }
+
```

Comment:
@atmgrifter00 @m-akinc is there some analog to mixins we can use in CSharp to make mixing in these apis consistent? Or even just an interface to share?

Reply:
Sounds fair that an interface has limited benefit today. What about shared testing? Being able to share tests that verify the needed attributes for Grouping, etc were implemented?

Reply:
Seems like we would also need something like mixins to share tests. We've used a base class for anchor tests, but we wouldn't be able to pull in multiple sets of shared tests (i.e. fractional width and grouping) that way.

Reply:
Instead of mixing the tests into the current test file could there be a separate NimbleTableColumnAnchorGroupingTests.cs that extends a GroupingTests.cs abstract base class?

If the [Fact] attributes have to be defined in the leaf class with a trivial implementation that calls the base class method that seems fine.

I dunno, C# is a lame type system but sure there are some options ;)

Doesn't have to block this PR, it's tech debt for cleaning up duplication in tests and ensuring consistency in our bindings. Can file an issue for it to track it separately since this PR is already massive.

Reply:
The problem with a base class is that you can only derive from one base class, so we can't inherit tests for both the FractionalColumnWidthAPI and the GroupingColumnAPI. Still, I've created tech debt item #1212 to track this.

Reply:
Yep, that's why the proposal was adding additional files extending different base classes. 👍 Thanks for tracking it.